pyth/plugins/rtf15/reader.py

```diff
@@ -139,57 +149,57 @@ def parse(self):
                 # inside groups we don't care about anyway
                 continue
 
-            elif next == '\\':
+            elif nextbyte == b'\\':
                 control, digits = self.getControl()
                 self.group.handle(control, digits)
             else:
-                self.group.char(next)
+                self.group.char(nextbyte)  # within-group text
 
 
     def getControl(self):
-        chars = []
-        digits = []
-        current = chars
-        first = True
+        chars = []  # for output 1
```

Comment:
I spent hours with the various aspects of getControl. While doing so, I became confident wrt the semantics of next and first (so I gave them nicer names nextbyte and isfirst) but never became quite sure what all the possibilities are what digits and chars may mean/contain. This insecurity is reflected in my vague comments.
Yes, better ones are warranted.